conpot/protocols/kamstrup/meter_protocol/kamstrup_server.py

```diff
@@ -15,21 +15,18 @@
 # Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-import socket
 import binascii
+import logging
 import random
-import conpot
+import socket
 from gevent.server import StreamServer
 import gevent
 from conpot.helpers import chr_py3
 import conpot.core as conpot_core
 from conpot.protocols.kamstrup.meter_protocol import request_parser
 from conpot.protocols.kamstrup.meter_protocol.command_responder import CommandResponder
 from conpot.core.protocol_wrapper import conpot_protocol
-# import logging as logger
```

Comment:
You can leave these comments here. It's super useful when debugging :slightly_smiling_face:

Reply:
I **really** dislike commented out code. Commented out code tends to rot. It's clutter. It tends to break over time because it's not tested. If you notice it you don't know whether to maintain it or not: is it obsolete? Was it left there intentionally? Or unintentionally? 

Thus I tend to remove commented out code. With a merge request of this size that was likely a bad idea because the change actually just adds clutter (hah) for the reviewer.

Do you really want to keep such code though?

Reply:
Note that in this specific case, there was even a line below (former line 32) that was incompatible or at least redundant with the commented out code.

Reply:
@xandfury how about adding this code into a debug function instead?

Reply:
@srenfo I don't think so.It works pretty well for me. I don't it's going to rot ~ unless it is generating code smell which it won't since it's commented. Again that's just my opinion. Others are free to disagree.

@glaslos Sure. That might be a better solution.

Reply:
It just so happens that I can illustrate my point. :sunglasses: (This was not intentional but here we are.)

You may have noticed two rogue logging lines on every Conpot startup:
```
$ conpot -f --template default
WARNING:scapy.loading:Cannot read wireshark manuf database
WARNING:scapy.runtime:No route found for IPv6 destination :: (no default route?)

                       _
   ___ ___ ___ ___ ___| |_
  |  _| . |   | . | . |  _|
  |___|___|_|_|  _|___|_|
              |_|

  Version 0.6.0
  MushMush Foundation
2020-05-26 15:20:47,753 --force option specified. Using testing configuration
...
```
Those are (in part) caused by this line at the top of [ipmi_server.py](https://github.com/mushorg/conpot/blob/master/conpot/protocols/ipmi/ipmi_server.py#L38):
```py
logger.basicConfig(stream=sys.stdout, level=logger.DEBUG)
```
which I heavily suspect was left in there as a debugging aid, maybe accidentally. (The second reason for those two lines is that BACpypes 0.17.0 adds a root-level logger in its `debugging.py`. You need both for those lines to appear.)

The offending line was removed in the PR, though I admit this was an unintentional change when I gobbled up all the debugging code.

Re: "debugging function" - that is the intended purpose of `start_protocol.py`! This seems to be a common misunderstanding throughout the discussion here. I'll try to address it in a separate comment below.

To move forward with the PR I will need directions though: do you want me to restore this code, yes or no?